tasks/site.yml

```diff
@@ -137,6 +137,32 @@
     mode: '0644'
   tags: [ 'role::checkmk_server:multisite' ]
 
+- name: Wait for the site to be started
+  wait_for:
+    path: '{{ checkmk_server__site_home }}/{{ checkmk_server__multisite_config_path }}/wato/users.mk'
+    timeout: 60
+
+- name: Read local multisite users definition
+  command: 'sed "1,4d" {{ checkmk_server__site_home }}/{{ checkmk_server__multisite_config_path }}/wato/users.mk'
+  register: checkmk_server_register_users_mk
+  changed_when: False
+  always_run: True
+  tags: [ 'role::checkmk_server:multisite' ]
+
+- name: Set multisite users definition fact
+  set_fact:
+    checkmk_server_fact_local_users: '{{ checkmk_server_register_users_mk.stdout }}'
+  always_run: True
+  tags: [ 'role::checkmk_server:multisite' ]
+
+- name: Set local multisite user passwords
+  htpasswd:
+    path: '{{ checkmk_server__site_home }}/etc/htpasswd'
+    name: '{{ item }}'
+    password: '{{ checkmk_server__multisite_users[item]["password"] }}'
+  when: '"password" in checkmk_server__multisite_users[item]'
```

Comment:
Just a little style thing. I mostly saw `(`, `)` used around when conditions in the DebOps project.